.github/workflows/rust.yml

```diff
@@ -38,3 +39,14 @@ jobs:
     - run: rustup toolchain install ${{ matrix.rust_channel }}
     - run: rustup default ${{ matrix.rust_channel }}
     - run: cargo test --verbose ${{ matrix.features }}
+
+  fuzz:
+    runs-on: ubtuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - run: rustup toolchain install nightly
+    - run: rustup default nightly
+    - run: cargo install cargo-fuzz
+    - run: cargo fuzz build ops --sanitizer none
```

Comment:
Would there be any value to enabling ASan here (i.e., omitting `--sanitizer none`) since we're testing an allocator with unsafe layout bits?

Reply:
I can investigate doing this in a follow up. My experience has been that things can be funky when ASan doesn't know about your custom allocator methods, but I'll try to get it up and running.